frontend/src/pages/modelCatalog/components/ModelCatalogCard.tsx

```diff
@@ -71,10 +71,17 @@ export const ModelCatalogCard: React.FC<{ model: CatalogModel; source: string }>
           </Split>
         </StackItem>
         <StackItem isFilled>{model.description}</StackItem>
+        <StackItem>
+          <ModelCatalogLabels labels={model.labels} tasks={model.tasks} />
+        </StackItem>
       </Stack>
     </CardBody>
     <CardFooter>
-      <ModelCatalogLabels labels={model.labels} tasks={model.tasks} />
+      {(model.tasks ?? []).map((task, index) => (
+        <Label variant="outline" key={index}>
```

Comment:
why do we have `ModelCatalogLabels` above, plus manually putting the tasks here? doesn't `ModelCatalogLabels` handle both?

Reply:
also, why move the `ModelCatalogLabels` out of the footer?